### PR TITLE
Write meaningful text alternatives for images

### PR DESCRIPTION
**Pull request by slhenry**
[mild] suggest not using "describes" since that insinuates to too wordy alt text
[mild] not say "assign" 'cause writing task is to write it, but they might not be setting it in the code :)

the full text would be:
... that provides the information presented by the image or the function of the image.
I think can shorten it as I've suggested.

### CODE REVIEW

**Comment by iadawn** [comment]
Incorporated changes from first sentence and made slight modification to second sentence to further suggest that this is about writing.

**Comment by davidberman** [comment]
I think that that solves the issue well.
